lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1350 @@
+      - The maximum time allowed to complete the healthcheck before an interval
+        is considered failed. Like start-period, the value can be expressed in
+        a time format such as 1m22s. The default value is 30s
+    type: str
+  hostname:
+    description:
+      - Container host name. Sets the container host name that is available inside the container.
+    type: str
+  http_proxy:
+    description:
+      - By default proxy environment variables are passed into the container if
+        set for the podman process. This can be disabled by setting the http_proxy
+        option to false. The environment variables passed in include http_proxy,
+        https_proxy, ftp_proxy, no_proxy, and also the upper case versions of those.
+        Defaults to true
+    type: bool
+  image_volume:
+    description:
+      - Tells podman how to handle the builtin image volumes.
+        The options are bind, tmpfs, or ignore (default bind)
+    type: str
+    choices:
+      - 'bind'
+      - 'tmpfs'
+      - 'ignore'
+  init:
+    description:
+      - Run an init inside the container that forwards signals and reaps processes.
+    type: str
+  init_path:
+    description:
+      - Path to the container-init binary.
+    type: str
+  interactive:
+    description:
+      - Keep STDIN open even if not attached. The default is false.
+        When set to true, keep stdin open even if not attached.
+        The default is false.
+    type: bool
+  ip:
+    description:
+      - Specify a static IP address for the container, for example '10.88.64.128'.
+        Can only be used if no additional CNI networks to join were specified via
+        'network:', and if the container is not joining another container's network
+        namespace via 'network container:<name|id>'.
+        The address must be within the default CNI network's pool (default 10.88.0.0/16).
+    type: str
+  ipc:
+    description:
+      - Default is to create a private IPC namespace (POSIX SysV IPC) for the container
+    type: str
+  kernel_memory:
+    description:
+      - Kernel memory limit (format <number>[<unit>], where unit = b, k, m or g)
+    type: str
+  label:
+    description:
+      - Add metadata to a container, pass dictionary of label names and values
+    type: dict
+  label_file:
+    description:
+      - Read in a line delimited file of labels
+    type: str
+  log_opt:
+    description:
+      - Logging driver specific options. Used to set the path to the container log file.
+        For example log_opt "path=/var/log/container/mycontainer.json"
+    type: str
+    aliases:
+      - log_options
+  memory:
+    description:
+      - Memory limit (format 10k, where unit = b, k, m or g)
+    type: str
+  memory_reservation:
+    description:
+      - Memory soft limit (format 100m, where unit = b, k, m or g)
+    type: str
+  memory_swap:
+    description:
+      - A limit value equal to memory plus swap. Must be used with the -m (--memory) flag.
```

Comment:
This is not helping the user, since there are no flags in Ansible. Do you mean the `memory` option?